components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -11,6 +12,7 @@
     model_validator,
     PlainSerializer,
     PrivateAttr,
+    SecretStr,
```

Comment:
this seems unused? did we attempt using this type for passwords? (and why do we end up not using it?)

Reply:
i see. if it's unused, let's remove it then. i think `task lint:fix-py` could have helped fixing this too

```suggestion
```